test/conformance/chainsaw/cli/with-namespace/policy.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: preconditions
+  //there is no namespace
```

Comment:
`ClusterPolicy` resources cannot have a `namespace` field because they are cluster scoped. Please test to ensure that even if this resource were written with a `namespace` that testing will still fail. A user cannot specify a `policy` field in `results[]` as `<namespace>/<policy_name>` if the kind is `ClusterPolicy`.

Reply:
Changed